.github/workflows/CI.yml

```diff
@@ -111,58 +99,41 @@ jobs:
           echo "${OE_LICENSE_TEXT}" > ${OE_LICENSE}
 
       - name: Install package
-        shell: bash -l {0}
         run: |
           conda remove --force openff-toolkit openff-toolkit-base
           python setup.py develop --no-deps
 
-      - name: Check installed toolkits
-        shell: bash -l {0}
+      - name: Remove undesired toolkits
         run: |
-          if [[ "$OPENEYE" == true ]]; then
-            python -c "from openff.toolkit.utils.toolkits import OPENEYE_AVAILABLE; assert OPENEYE_AVAILABLE, 'OpenEye unavailable'"
-          fi
-          if [[ "$OPENEYE" == false ]]; then
-            if [[ $(conda list | grep openeye-toolkits) ]]; then
-              conda remove --force openeye-toolkits --yes
-            fi
-            python -c "from openff.toolkit.utils.toolkits import OPENEYE_AVAILABLE; assert not OPENEYE_AVAILABLE, 'OpenEye unexpectedly found'"
-          fi
-          if [[ "$RDKIT" == true ]]; then
-            python -c "from openff.toolkit.utils.toolkits import RDKIT_AVAILABLE; assert RDKIT_AVAILABLE, 'RDKit unavailable'"
-          fi
-          if [[ "$RDKIT" == false ]]; then
-            if [[ $(conda list | grep rdkit) ]]; then
-              conda remove --force rdkit --yes
-            fi
-            if [[ $(conda list | grep ambertools) ]]; then
-              conda remove --force ambertools --yes
-            fi
-            python -c "from openff.toolkit.utils.toolkits import RDKIT_AVAILABLE; assert not RDKIT_AVAILABLE, 'RDKit unexpectedly found'"
+          if [[ ${{ env.PACKAGES_TO_REMOVE }} ]]; then
+            for cpkg in ${{ env.PACKAGES_TO_REMOVE }}; do
+              echo $cpkg
+              if [[ $(conda list | grep $cpkg) ]]; then conda remove --force $cpkg --yes ; fi
+            done
           fi
 
+      - name: Check installed toolkits
+        run: |
+          for tk in ${{ env.TOOLKIT_CHECKS }}; do
+            python -c "from openff.toolkit.utils.toolkits import ${tk}_AVAILABLE; assert ${tk}_AVAILABLE, '${tk} unavailable'"
+          done
+
```

Comment:
If I'm reading this right, it checks that the desired toolkits are installed (behavior that seems to be working) but does not explicitly check that that undesired toolkits are **not** installed? This may seem like an obtuse use case but for testing some OpenEye behavior, we want to make sure `OpenEyeToolkitWrapper` is being used.

Reply:
Added!